README.md

```diff
@@ -6,22 +6,140 @@ This project was generated with [Angular CLI](https://github.com/angular/angular
+
+### Usage
+
+The filter provides several properties according to which log messages can be filtered. Each of this properties has default value in the case that user does not set its value. After clicking **Filter** button, log messages are displayed in table under the filter.
+
+## Users
+
+It is possible to perform CRUD operation over MySQL tables, where information about users are stored. Currently there are two tables available:
+
+-   **approved users**
+-   **users waiting for approval**
+
+### Usage
+
+The users tab provides CRUD functionality over both tables. Also there is extra approval functionality for users, that are stored in **users waiting for approval** table. After clicking on green check icon, modal window with the form is displayed. To successfully validate the user, it is necessary to fill in either SDO or Vendor access rights.
+
+## Config
+
+The config tab simply displays the content of the configuration file, which contains the individual configuration details of the yangcatalog.
+
+### Usage
+
+Users with the permissions granted have the option to update the content of the configuration file by clicking on the **Edit** button. After applying the changes, it is necessary to click the **Save** button for saving changes.
+
+## Nginx
+
+The NGINX tab displays the content of the NGINX files. There are 3 NGINX files which can be displayed or edited.
+
+### Usage
+
+This tab works exactly like a config tab file, except that it is necessary to first select which NGINX file you want to display.
+
+## Files
+
+The files tab provide reading, updating and deleting content of files from main yangcatalog folder. The information contained in this tab can be divided into two parts:
+
+-   **Pie charts** - Charts display total disk usage and disk usage of selected folder
+-   **Directory structure** - List of folders and corresponding files and sub-folders
+
+### Usage
+
+Displayed pie charts are mainly for informational purposes only, it is possible to show disk usage of individual folders on hover at corresponding slices in chart. Second pie chart redraws after each click on the folder in directory structure.
+
+Directory structure displays information about the size and permission of each file and folder. After clicking on user icon after name of file/folder permissions are displayed in modal window in more human-readable format. Files can be edited in the same way as config or NGINX files. It is also possible to delete files for users with the permissions granted.
+
+## Actions
+
+Last tab allows users to manually run some of the scripts that can be selected from the list. Each script has its own arguments, which can be set by user. Script itself and also each of the arguments has its own help message to help the user better understand their meaning.
+List of scripts that can be executed:
 
-Run `ng test` to execute the unit tests via [Karma](https://karma-runner.github.io).
+-   **populate**
```

Comment:
Please provide brief description (one or two sentences to each one). if you are not sure what it does send me a PM. elkRecovery say it is deprecated since we are using AWS service for elasticsearch